### PR TITLE
[ZRH-2021] Add program and speakers

### PR DESCRIPTION
**Pull request by RomanoRoth**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: data/events/2021-zurich.yml [hunk 1]
```diff
@@ -41,8 +41,8 @@ nav_elements: # List of pages you want to show up in the navigation of your page
   - name: registration
   - name: diversity
   - name: covid
- # - name: program
- # - name: speakers
+#  - name: program
```
Comment (glasnt) [inline data/events/2021-zurich.yml:44]:
```suggestion
  - name: program
```

File: data/events/2021-zurich.yml [hunk 2]
```diff
@@ -41,8 +41,8 @@ nav_elements: # List of pages you want to show up in the navigation of your page
...
+#  - name: speakers
```
Comment (glasnt) [inline data/events/2021-zurich.yml:45]:
```suggestion
  - name: speakers
```

**Comment by glasnt** [comment]
Two comments: 
 * you should remove the `Propose` links if your CFP is now closed
 * your speaker pictures are very large and take a while to load. I would suggest making them smaller in filesize and resolution

File: data/events/2021-zurich.yml (hunk 1 above)
Comment (RomanoRoth) [reply data/events/2021-zurich.yml:44]: Hi @glasnt 
Thanks for the suggestion. But at the moment we don't want to release/publish the program and the speakers yet.  Therefore we just deploy them allready. (I know that you can access them when you know the URL).

File: data/events/2021-zurich.yml (hunk 2 above)
Comment (RomanoRoth) [reply data/events/2021-zurich.yml:45]: Hi @glasnt 
Thanks for the suggestion. But at the moment we don't want to release/publish the program and the speakers yet.  Therefore we just deploy them allready. (I know that you can access them when you know the URL).

**Comment by RomanoRoth** [comment]
@glasnt 
Thanks for the two comments. 
If resized the pictures and removed the "propose" link.

**Pull request merged**